eclipse-collections-code-generator/src/main/resources/impl/map/mutable/objectPrimitiveHashMap.stg

```diff
@@ -426,6 +427,18 @@ public class Object<name>HashMap\<K> implements MutableObject<name>Map\<K>, Exte
         map.forEachKeyValue(this::put);
     }
 
+    @Override
+    public void updateValues(Object<name>To<name>Function\<? super K> update)
+    {
+        for (int i = 0; i \< this.keys.length; i++)
+        {
+            if (isNonSentinel(this.keys[i]))
+            {
+                this.values[i] = update.valueOf(this.toNonSentinel(this.keys[i]), this.values[i]);
+            }
+        }
```

Comment:
I see that this is not the analog of the method MutableMap.updateValue(). That method takes a key and applies to a single entry. This method is closer to collectValues(), but in-place. I'm not sure about the name.

Reply:
@motlin I like `updateValues`, but I understand what you mean. I don't know, maybe someone else would have a useful opinion?